charts/andi/values.yaml

```diff
@@ -88,7 +95,7 @@ theme:
 footer:
   # footer text on left side with links on right side
   leftSideText: "Some Left Side Text"
-  links: "[{\"linkText\":\"Example 1\", \"url\":\"https://www.example.com\"}, {\"linkText\":\"Example 2\", \"url\":\"https://www.google.com\"}]"
+  links: '[{"linkText":"Example 1", "url":"https://www.example.com"}, {"linkText":"Example 2", "url":"https://www.google.com"}]'
```

Comment:
Confirmed in the template results that removing these escape characters made no alteration to the outcome value.